code/modules/cargo/exports/large_objects.dm

```diff
@@ -15,14 +15,21 @@
 
 /datum/export/large/crate/wooden
 	cost = 100
-	unit_name = "wooden crate"
+	unit_name = "large wooden crate"
 	export_types = list(/obj/structure/closet/crate/large)
 	exclude_types = list()
 
 /datum/export/large/crate/wooden/ore
 	unit_name = "ore box"
 	export_types = list(/obj/structure/ore_box)
 
+/datum/export/large/crate/wood
+	cost = 150
```

Comment:
Shouldn't this be at least 240, given 40/plank and 6 planks/crate ?

A small premium could be fair, since these also take time to build.